.chloggen/internal-metrics-update.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: collector
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Updates the otel internal metrics receiver to drop grpc/http request and response metrics (new with v0.93.0).
+# One or more tracking issues related to the change
+issues: [1146]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
I don't think we need a change log, though. There is nothing really fixed from the previous version

Reply:
ack, thought it was better to make a note in the changelog